user_tools/src/spark_rapids_pytools/cloud_api/dataproc.py

```diff
@@ -360,13 +369,23 @@ def extract_gpu_name(gpu_description: str) -> str:
     def get_instance_description_cli_params(self) -> list:
         return ['gcloud compute machine-types list', '--zones', f'{self.get_zone()}']
 
+    def init_instance_descriptions(self) -> None:
```

Comment:
The implementation seems to be similar to `emr` platform. Can we move this to the base class to avoid duplication?

Reply:
Yes, I am planning to move this to base class after we finish with all the platforms. I am planning to put a final PR to clean up.